src/test/java/com/ksm/wstbackoffice/controller/CountryControllerIT.java

```diff
@@ -56,4 +72,16 @@ public void findAllTest() {
             statusCode(200).
             body("size()", equalTo(5));
     }
+
+    @ParameterizedTest
+    @MethodSource("findCountriesStartWithTestArguments")
+    public void findCountriesStartWith(List<String> stringsToStartWith, int expectedStatus) {
+        given().
+            contentType(ContentType.JSON).
+            body(stringsToStartWith).
+        when().
+            post("countries/start-with").
+        then().
+            statusCode(expectedStatus);
+    }
```

Comment:
I'd like you to add another test.
Some of the countries contain more than one word. E.g. United States of America or North Korea.
I want your algorithm to find
North Korea
When you specify "kor"
And to find 
United States of America
when you specify "amer"
Or to find
Democratic Republic of the Congo
when you specify "co"

Reply:
done